README.md

```diff
@@ -620,6 +620,20 @@ They are based on these older repositories:
 
 - [EF Core Power Tools](https://marketplace.visualstudio.com/items?itemName=ErikEJ.EFCorePowerTools&ssr=false#overview) - Visual Studio extension that can generate an Entity Framework Core DbContext and model classes, and various diagrams directly from your MSBuild.Sdk.SqlProj project.
 
+## .NET runtime support
+
+MSBuild.Sdk.SqlProj requires a .NET SDK to be installed. Depending on the version of MSBuild.Sdk.SqlProj you can use varying versions of the .NET SDK (released once a year). The following table lists the supported .NET SDK versions by each MSBuild.Sdk.SqlProj version:
+
+| SDK version  | .NET runtime version |
```

Comment:
Since we're now talking about "a .NET SDK", I find "SDK version" here to be confusing. I'd recommend that we call this column "MSBuild.Sdk.SqlProj version"

Reply:
@jeffrosenberg Thanks, good feedback!